default.nix

```diff
@@ -168,16 +168,16 @@ rec {
       ln -s ${compressedJs frontend} $out/frontend.jsexe
     ''; #TODO: run frontend.jsexe through the asset processing pipeline
 
-  server = exe: hostName:
+  server = exe: hostName: sslHost: adminEmail:
```

Comment:
I don't think we should just add more params here.  Let's make it use a record or something instead.